### PR TITLE
Features work Youtube page 

### PR DESCRIPTION
**Pull request by Fanger53**
this work contains a clone of youtube homepage made by 2 collaborators, the languages using in this project was HTML and CSS. the workflow you can follow it in https://github.com/BrianSammit/Youtube-page .  before the project was rejected because the repository was invalid so they couldn't review it and that was the reason to make another repository.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+              <a href="">
+                <i class="fas fa-thumbs-up"></i>
+              </a>
+              <p class="info-p">5560</p>
+              <a href="">
+                <i class="fas fa-thumbs-down"></i>
+              </a>
+              <p class="info-p">197</p>
+              <a href="">
+                <i class="fas fa-share"></i>
```
Comment (HADDADSOHAIB) [inline index.html]: The share icon should be a button:
- [ ] It has a share button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

File: index.html [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="css/flexboxgrid.min.css">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css"
+    integrity="sha384-xxzQGERXS00kBmZW/6qxqJPyxW3UR0BPsL4c8ILaIWXva5kFi7TxkIIaMiKtqV1Q" crossorigin="anonymous">
+  <link rel="shortcut icon" type="image/x-icon" href="img/favicon.ico">
+</head>
+
+<body>
+  <nav></nav>
```
Comment (HADDADSOHAIB) [inline index.html]: Don't leave unused tags in your html, this is a bad practice, it clutters your code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+          <div id="chanel-description">
+            <h4>Sign up for my email newsletter of the best motorcycling stories here:</h4>
+            <a href="https://mailchi.mp/306039bd7519/motobob">
+              <span>https://mailchi.mp/306039bd7519/motobob</span>
+            </a>
+            <p class="info-p"><b><br><b><br>SHOW MORE</p>
+          </div>
+        </div>
+        <hr>
+        <b><br><b><br>
```
Comment (HADDADSOHAIB) [inline index.html]: This is not a correct use of `<b>` tag, we use `<b></b>` to make a text bold and it should have a closing tag. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b. Please select only the text that you want to highlight and not the whole sections of HTML code.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+                  <div id="box-comment">
+                    <input id="commets-bar" type="text" name="comment" placeholder="Add a public comment...">
+                  </div>
+                  <div id="cc-button">
+                    <div id="button-cancel">CANCEL</div>
+                    <div id="button-comment">COMMENT</div>
+                  </div>
+                </div>
+              </div>
+              <b><br><b><br><b><br><b><br>
```
Comment (HADDADSOHAIB) [inline index.html]: - [Optional] Instead of repeating `<br>` repeatedly, use margin and padding in css files.

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change required :recycle: 

### Hi @Fanger53  and @BrianSammit  :hand: 

### Great job with the project, you did great work. There are some changes to be made to approve the project:
- check my comments for additional requirements.
- In the project description, there was a link that you should have used to build your project:
image
http://archive.vn/Bss88
there a slight different between your project and that link in the section under the main video.

 | Source Page	| Clone Page |
|:---:|:---:|
|Screenshot from 2020-07-10 11-51-12 |Screenshot from 2020-07-11 00-27-18 |

- Good practice in git is to commit as much as possible and to commit atomic changes not the whole project at once, please make sure to do that from here further.
- [Optional] use the `<nav><\nav>` tag for your navbar.
- [Optional] use the `<main><\main>` tag for the main part of the HTML page.
### Once done with the changes, ask for another code review.

**Comment by BrianSammit** [comment]
We made the changes from the last pull request.
we adding a margin instead of using <br> tags to make space between objects and we deleted the <nav> tag from the HTML.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## Hello team 👋, great job making the changes. Really awesome project 💯 !

### However, we still need to work on some more change before we can merge.

- As requested by the previous reviewer, the section under the main video is not the same as the original page. Use this link as the source: https://archive.vn/Bss88

| Source Page	| Clone Page |
|:---:|:---:|
|Annotation 2020-07-13 231014|Annotation 2020-07-11 155039|

- The `span.slider round` is overlapping with the navbar. Please fix this.

image

Apply these changes and request another review.

Happy coding 👨‍💻

**Comment by BrianSammit** [comment]
we fixed the video Info part and we fixed the slider button from the recommended video.

**Comment by krys2fa** [comment]
## Status: Changes Required ♻️ 

Awesome job implementing the changes requested, team 👏 
The project is really taking shape nicely. Kindly work on the following to get the project approved.

- [ ] Please include the footer section as seen in the archive. Use this link as the source: Use this link as the source: [Archive link](https://archive.vn/Bss88)

Screen Shot 2020-07-13 at 9 22 48 PM


- [ ] [OPTIONAL] The use of semantic HTML tags like `<main>` and [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) are highly encouraged.

- [ ] [OPTIONAL] Please update the link to the live version of your project in the README.md file to ensure that the latest updates to your project can be easily seen.

Apply these changes and request another code review. 
Feel free to reach out for any clarification or assistance.

Happy Coding 🖥️  😃 
@ Chris Amanor [slack]

**Comment by Fanger53** [comment]
adding footer and update live demo from Readme

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## Impressive progress so far 💪! Thanks for making all the changes 👏.

### However, we need to work on some changes before we can merge.

While applying changes to the footer, you have introduced a horizontal scrollbar 👇 

image

I see that you have set the width of the footer as 100vw and also margins on all sides. Remember the box-model here. Margins are not part of the content. So right now, your footer's width is 100vw+(25px+25px), which is greater than any screen width. That's why the overflow. 


Your project looks absolutely fantastic. Just make this change and apply for another code review.

Happy coding 👨‍💻

**Comment by BrianSammit** [comment]
we change the width footer.

**Comment by krys2fa** [comment]
# Status: Project Approved! 🏁 
Congratulations on making all the requested changes and getting this project approved. 👍 

You can now merge this branch into the master branch and go on with your next project.

💡  Do not forget to submit your project completion form after merging this branch.

All the best team,
Chris

**Pull request merged**